src/components/OrganizationListTablePanel.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { OrganizationBundle } from '@pdc/sdk';
+import { Panel, PanelBody } from './Panel';
+import { OrganizationListTable } from './OrganizationListTable';
+
+interface OrganizationListTablePanelProps {
+	organizations?: OrganizationBundle;
+	loading?: boolean;
+}
+
+const DEFAULT_COLUMNS = ['Organization Name', 'Organization EIN'];
```

Comment:
I think we can take out `Organization` from both column names (but at least the `EIN` column name)